media/javascript/elastic-search.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function () {
```

Comment:
Visto che `Qs` non è chiaro dove sia definito, metterei almeno un commento in testa al file. Se si potesse evitare l'utilizzo, meglio (la stringa a riga 12 si può creare con [jQuery param[(https://api.jquery.com/jQuery.param/), l'altra è da vedere).

Reply:
Avevo fatto qualche ricerca per evitare di usare Qs ma non c'è modo di fare il procedimento inverso. Volevo usare una riduzione e concatenare le stringhe ma non è possibile usarla nativamente sugli oggetti.

Aggiungo comunque il commento.